### DISCUSSION TITLE
Seeding production db

### DISCUSSION THREAD

**Discussion by felixyeboah** [epicweb-dev/epic-stack#631] [General]
i’ve been going through the documentation for seeding production database and i don’t know if i’m doing something wrong. i followed the process of creating another script similar to `prisma/seed.ts`. after running the script and also running the dump script, the dump sql file seems empty. there’s nothing in that file.

also, i want to state, i have multiple migration files due to additions made to the schema file and i don't know if thats the cause of the issue.

any help with this please 🙏

**Comment by kentcdodds** [comment]
Hi @felixyeboah,

It's hard to say what's wrong without seeing an example of the exact steps you took because you probably missed a step or got something wrong.

**Reply by felixyeboah** [reply]
![CleanShot 2024-03-01 at 00 06 05@2x](https://github.com/epicweb-dev/epic-stack/assets/87368189/b87a8746-d694-4aba-af88-78fe7a6d02aa)

Hi @kentcdodds, 

sorry for the late reply. I attached a screenshot of what I see within the `seed.sql` file. I followed the steps in the doc and I have tried several times to replicate what's in the doc but nothing seems to work.

**Reply by kentcdodds** [reply]
I believe you when you say it's empty. Could you list the steps with the exact commands and code you're running?

**Reply by felixyeboah** [reply]
okay got it.

- created a similar seed script with the items i want seeded
- run the script to generate the data locally `prisma db seed`
- created a dump of the seed db using the script in the docs `sqlite3 seed.db .dump > seed.sql `

After the script, it generated 2 files, `seed.db` and `seed.sql`, and both files came out empty

**Reply by kentcdodds** [reply]
I'm pretty sure you're seeding a different database from the one you're generating the seed.sql with.

**Reply by felixyeboah** [reply]
okay, if I get you right, I have the seed of the database in the Prisma folder? `data.db` file in the Prisma folder

**Reply by kentcdodds** [reply]
Sorry, to be more clear, the step you got wrong is the "run your seed script" step. That's probably my fault because it's not clear how to do this. You don't run `prisma db seed` to run your custom script and have it pointing to a custom database.

I've updated the docs to be more clear: https://github.com/epicweb-dev/epic-stack/blob/main/docs/database.md

Thanks for your patience and persistence!

**Reply by felixyeboah** [reply]
this is clear. got it now. thank you so much for the help. God bless you

**Reply by felixyeboah** [reply]
@kentcdodds I have one more question based on this step:

5. Copy the relevant bits from the seed.local.sql file into your migration.sql file. The seed.local.sql will include create table/index lines etc. which should already be in your migration.sql. You probably just want INSERT commands.

my question: say I have multiple migration.sql files, which of them do I include the INSERT commands? is it the first migration.sql file or the last migration.sql file?

**Reply by kentcdodds** [reply]
Put them wherever it makes sense. Probably the last one.

**Reply by felixyeboah** [reply]
thanks
